<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>6</pointsize>
   </font>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MES Logiciel</string>
  </property>
  <property name="windowIcon">
   <iconset resource="data/data.qrc">
    <normaloff>:/icons/icons/icons8-production-100.png</normaloff>:/icons/icons/icons8-production-100.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(250, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>511</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" rowspan="2">
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">
border-image: url(:/icons/icons/icons8-production-100.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="text">
        <string>MES-Logiciel</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>copyright</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>198</y>
      <width>583</width>
      <height>205</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Entrer le nombre de job :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(213, 213, 213);</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QTableWidget" name="table">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(227, 227, 227);</string>
       </property>
       <row>
        <property name="text">
         <string>M1</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>M2</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>J1</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>J2</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>J3</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="calcule">
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
    background-color: white; /* Couleur de fond blanche */
    border: 2px solid #4CAF50; /* Bordure verte */
    color: black; /* Couleur du texte noire */
    padding: 10px 20px; /* Espacement interne */
    font-size: 16px; /* Taille de la police */
    border-radius: 20px; /* Coins arrondis */
    min-width: 100px; /* Largeur minimale du bouton */
    transition: all 0.3s ease; /* Transition douce pour toutes les propriétés */
}

QPushButton:hover {
    background-color: #f0f0f0; /* Légère couleur grise au survol */
}

QPushButton:pressed {
    background-color: #e0e0e0; /* Couleur gris clair lors du clic */
    padding-left: 14px; /* Effet de &quot;pression&quot; */
    padding-top: 14px; /* Le bouton semble bouger vers le bas */
}</string>
       </property>
       <property name="text">
        <string>calcule</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="create">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>create</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>414</y>
      <width>577</width>
      <height>64</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionNouveau"/>
    <addaction name="actionopen_2"/>
    <addaction name="actionFermer"/>
    <addaction name="separator"/>
    <addaction name="actionEnregistrer"/>
    <addaction name="actionEnregistrer_sous"/>
    <addaction name="separator"/>
    <addaction name="actionquiter"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuabout">
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="title">
     <string>About</string>
    </property>
    <widget class="QMenu" name="menuAlgorithme">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="title">
      <string>Algorithme</string>
     </property>
     <addaction name="actionJohnson_s_algorithme"/>
    </widget>
    <addaction name="menuAlgorithme"/>
    <addaction name="separator"/>
    <addaction name="actionMES_Logiciel"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="geometry">
     <rect>
      <x>217</x>
      <y>113</y>
      <width>153</width>
      <height>158</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actioncouper"/>
    <addaction name="actioncopier"/>
    <addaction name="actioncoller"/>
    <addaction name="actionsupprimer"/>
    <addaction name="separator"/>
    <addaction name="actionFind"/>
   </widget>
   <widget class="QMenu" name="menufenetre">
    <property name="font">
     <font>
      <pointsize>8</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Fenetre</string>
    </property>
    <addaction name="actionfall_screan"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuEdit"/>
   <addaction name="menufenetre"/>
   <addaction name="menuabout"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNouveau"/>
   <addaction name="actionopen_2"/>
   <addaction name="actioncouper"/>
   <addaction name="actioncopier"/>
   <addaction name="actioncoller"/>
   <addaction name="actionsupprimer"/>
   <addaction name="actionFind"/>
  </widget>
  <action name="actionabout">
   <property name="text">
    <string>about</string>
   </property>
  </action>
  <action name="actionNouveau">
   <property name="icon">
    <iconset resource="data/data.qrc">
     <normaloff>:/icons/icons/icons8-fichier-40.png</normaloff>:/icons/icons/icons8-fichier-40.png</iconset>
   </property>
   <property name="text">
    <string>Nouveau...</string>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <stylestrategy>NoAntialias</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="actionJohnson_s_algorithm_2">
   <property name="text">
    <string>Johnson's algorithm</string>
   </property>
   <property name="font">
    <font>
     <pointsize>6</pointsize>
    </font>
   </property>
  </action>
  <action name="actionJohnson_s_algorithme">
   <property name="text">
    <string>Johnson's algorithme</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
  <action name="actionopen">
   <property name="text">
    <string>open</string>
   </property>
   <property name="font">
    <font>
     <pointsize>6</pointsize>
    </font>
   </property>
  </action>
  <action name="actionopen_2">
   <property name="icon">
    <iconset resource="data/data.qrc">
     <normaloff>:/icons/icons/icons8-dossier-ouvert-48.png</normaloff>:/icons/icons/icons8-dossier-ouvert-48.png</iconset>
   </property>
   <property name="text">
    <string>Ouvrir...</string>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionFermer">
   <property name="text">
    <string>Fermer</string>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="shortcut">
    <string>Ctrl+F4</string>
   </property>
  </action>
  <action name="actionEnregistrer">
   <property name="icon">
    <iconset resource="data/data.qrc">
     <normaloff>:/icons/icons/icons8-sauvegarder-48.png</normaloff>:/icons/icons/icons8-sauvegarder-48.png</iconset>
   </property>
   <property name="text">
    <string>Enregistrer</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
  <action name="actionEnregistrer_sous">
   <property name="icon">
    <iconset resource="data/data.qrc">
     <normaloff>:/icons/icons/icons8-enregistrer-sous-48.png</normaloff>:/icons/icons/icons8-enregistrer-sous-48.png</iconset>
   </property>
   <property name="text">
    <string>Enregistrer sous</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
  <action name="actionquiter">
   <property name="text">
    <string>Quitter</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
  <action name="actioncouper">
   <property name="icon">
    <iconset resource="data/data.qrc">
     <normaloff>:/icons/icons/icons8-couper-48.png</normaloff>:/icons/icons/icons8-couper-48.png</iconset>
   </property>
   <property name="text">
    <string>couper</string>
   </property>
  </action>
  <action name="actioncopier">
   <property name="icon">
    <iconset resource="data/data.qrc">
     <normaloff>:/icons/icons/blue-document-copy.png</normaloff>:/icons/icons/blue-document-copy.png</iconset>
   </property>
   <property name="text">
    <string>copier</string>
   </property>
  </action>
  <action name="actioncoller">
   <property name="icon">
    <iconset resource="data/data.qrc">
     <normaloff>:/icons/icons/icons8-coller-comme-texte-48.png</normaloff>:/icons/icons/icons8-coller-comme-texte-48.png</iconset>
   </property>
   <property name="text">
    <string>coller'</string>
   </property>
  </action>
  <action name="actionsupprimer">
   <property name="icon">
    <iconset resource="data/data.qrc">
     <normaloff>:/icons/icons/cross-script.png</normaloff>:/icons/icons/cross-script.png</iconset>
   </property>
   <property name="text">
    <string>supprimer</string>
   </property>
  </action>
  <action name="actionMES_Logiciel">
   <property name="text">
    <string>MES-Logiciel</string>
   </property>
  </action>
  <action name="actionfall_screan">
   <property name="text">
    <string>fall screan</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
  <action name="actionFind">
   <property name="icon">
    <iconset resource="data/data.qrc">
     <normaloff>:/icons/icons/icons/binocular.png</normaloff>:/icons/icons/icons/binocular.png</iconset>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="data/data.qrc"/>
 </resources>
 <connections/>
</ui>
